<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faul_menu</class>
 <widget class="QDialog" name="faul_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QPushButton" name="bt_ft_ko">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="bt_close">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="bt_change_on_faul">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="bt_foul_done">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="bt_oreb">
     <property name="text">
      <string>OREB</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="bt_foul_recv">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="bt_ft_ok">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lb_ft_perc">
     <property name="text">
      <string>FT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fouls Rec/Done</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="cb_MyTeam"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cb_Faul"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cb_OthTeam"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
